src/app.html

```diff
@@ -1,6 +1,125 @@
 <template>
-    <require from="./css/styles.css"></require>
-    <require from="./commons/navbar-searchbar/navbar-searchbar"></require>
-    <navbar-searchbar></navbar-searchbar>
-    <router-view class=""></router-view>
```

Comment:
why are you deleting this and overwriting this file? `app.html` file is our main file for navbar, footer and placeholder for router. If you are creating a component, you need to do this in a separate package, creating corresponding .html and .js files(like `task-list.html` and `task-list.js`. then adding a router to the component in `app.js` file:

`
 { route: 'new-task', moduleId: PLATFORM.moduleName('task/task-edit'), name: 'new-task' },
`